codalab/model/tables.py

```diff
@@ -348,7 +336,10 @@
   Column('user_id', String(63), ForeignKey(user.c.user_id), primary_key=True, nullable=False),
   Column('worker_id', String(127), primary_key=True, nullable=False),
 
-  Column('slots', Integer, nullable=False),  # Number of additional bundles the worker can run.
+  Column('tag', Text, nullable=True),  # Tag that allows for scheduling runs on specific workers.
```

Comment:
What if a user, for some weird reason, specifies a tag longer than 63 characters and then tries to use that tag in the --request-queue flag?